R/aggregate_to_epoch.R

```diff
@@ -0,0 +1,21 @@
+aggregate_to_epoch = function(ts, to, lightpeak_available, params_general) {
+  
+  # Aggregate time
+  ts$time_num = floor(as.numeric(iso8601chartime2POSIX(ts$time,tz = params_general[["desiredtz"]])) / to) * to
```

Comment:
Timestamp format conversion should not be here. The name of the function only refers to aggregation to epoch and we risk doing the same conversion multiple times.  I think this should be done centrally inside for example g.part5_initialise_ts.